prerequisites.rst

```diff
@@ -1,35 +1,250 @@
+operating system that is Debian. The downloable ISO image ready to use from
+the oficial web page only supports at the moment amd64 with BIOS mode boot.
+
+The System drive storage (SDS)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The system storage disk will be fully managed and taken over by the |omv| after
+installed. Any brand is supported as is also supported by the Debian system.
+For Flash Drive type storage devices as well as solid state drives being used as
+system storage drives please refer to the last section for technical details.
+
+In case of the official ISO instalation, will automatically partition the system
+storage disk drive into 3 partitons. Consult the next table for.
+
+In case of a manual installation on a previous Debian operating system, this 
+**drive must have at least two partitions**. Consult the next table for.
+
+============  ==========  ===========  =======================================
+ Partition     Min size    Best size    Mandatory
+============  ==========  ===========  =======================================
+ ``/boot``      256Mib      500Mib      Optional, partition used to boot
+ ``/``           4Gi        120Gib      Yes, the partition were system install
+ ``swap``       100Mib      16Gib       Yes, the partition for virtual ram
+============  ==========  ===========  =======================================
+
+The Data drive storage (DDS)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The data storage disk will be partially managed by the |omv| after installed,
+and only in customized installation can be a partiton of the same system drive
+storage.
+
+Each data storage drive will be managed, but regardless of the partitioning
+scheme, all data drive partitions will be mounted at the ``/srv/`` path.
+The **data storage drive must be a different disk drive than the system disk drive**
+where the operating system is installed. **One partition of each data
+drive can handle one or many shared resources.**
+
+==============  ==========  ===========  ========================================
+ Partition       Min size    Best size    Mandatory
+==============  ==========  ===========  ========================================
+ ``/srv/<*>``     100Mib       bigger     Yes, size as need! example 128Gib
+==============  ==========  ===========  ========================================
+
+Technical notes
+---------------
+
+Take into **consideration that as more exquisite the customization as less supported
+could be.**.
+
+|omv| allows for minimal deployment and use of the system with very minimal
+hardware/software requirements, at the cost of the key knowledge, but other NAS
+systems allow ease of use at the cost of high requirements. |omv| allows both as
+well in a well balanced use cases! [7]_
+
+About minimal or custom setups
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It can be noted that |omv| system can be installed on just 4Gib of system partition,
+with no more than 1Gib of RAM and using a WiFi or USB connection to access it
```

Comment:
```suggestion
with no more than 1GiB of RAM and using a WiFi or USB connection to access it
```

Reply:
fixed with commit c441ec7d9649da3fb60e9f23635c041b62f2a54c al Gib ocurrences changed to GiB